### PR TITLE
front: editor - using rtk in infra error modal

### PR DESCRIPTION
**Pull request by sim51**

### CODE REVIEW

File: editoast/openapi.yaml [hunk 1]
```diff
@@ -1722,3 +1752,24 @@ components:
...
+            - Point
+            - LineString
+            - Polygon
+            - MultiPoint
+            - MultiLineString
+            - MultiPolygon
+        coordinates:
+          type: array
+          items:
+            type: number
```
Comment (jacomyal) [inline editoast/openapi.yaml]: This is not correct:
- For `Point`, `coordinates` must be `number[]`
- For `MultiPoint` and `LineString`, `coordinates` must be `number[][]`
- For `MultiLineString` and `Polygon`, `coordinates` must be `number[][][]`
- For `MultiPolygon`, `coordinates` must be `number[][][][]`

**Review by jacomyal — DISMISSED** [review]

File: editoast/openapi.yaml [hunk 2]
```diff
@@ -420,7 +420,7 @@ paths:
                     nullable: true
                     example: null
                     description: The index of the previous page (null if requesting the first page)
-                  results:
+                  result:
```
Comment (flomonster) [inline editoast/openapi.yaml]: I checked on `dev` and the field name is `results`.

**Review by flomonster — CHANGES_REQUESTED** [review]

File: editoast/openapi.yaml (hunk 2 above)
Comment (sim51) [reply editoast/openapi.yaml]: Indeed, ... I need to rebuild the docker stack :)
let me fix it

File: editoast/openapi.yaml [hunk 3]
```diff
@@ -1722,3 +1752,124 @@ components:
...
+    Geometry:
+      type: object
+      description: Definition of a GeoJSON geometry
+      oneOf:
+        - $ref: "#/components/schemas/Point"
+        - $ref: "#/components/schemas/LineString"
+        - $ref: "#/components/schemas/Polygon"
+        - $ref: "#/components/schemas/MultiPoint"
+        - $ref: "#/components/schemas/MultiLineString"
+        - $ref: "#/components/schemas/MultiPolygon"
```
Comment (flomonster) [inline editoast/openapi.yaml:1765]:
```suggestion
      oneOf:
        - $ref: "#/components/schemas/Point"
        - $ref: "#/components/schemas/LineString"
        - $ref: "#/components/schemas/Polygon"
        - $ref: "#/components/schemas/MultiPoint"
        - $ref: "#/components/schemas/MultiLineString"
        - $ref: "#/components/schemas/MultiPolygon"
      discriminator:        
        propertyName: type
```

**Review by flomonster — APPROVED** [review]
Approved. We can add a discriminator though (it might generate something different with rtk) :shrug:.

File: editoast/openapi.yaml (hunk 3 above)
Comment (sim51) [reply editoast/openapi.yaml:1765]: With the discriminator syntax, the generation is a bit different, but the result at the end is the same :

With the discriminator :  ` export type Geometry =  ( {  type: 'Point';   } & Point ) | ...`
Without: `export type Geometry = Point |  ...`

And the Point type definition is the same, ie. 
```
export type Point = {
  type: 'Point';
  coordinates: Point3D;
};
```

So in this use-case, I don't see the utility of the discriminator syntax. Perhaps I miss something ...

**Review by jacomyal — APPROVED** [review]

**Pull request merged**